diff --git a/sgp4/tests.py b/sgp4/tests.py
index 618f786..f2ef91a 100644
--- a/sgp4/tests.py
+++ b/sgp4/tests.py
@@ -10,6 +10,7 @@ import re
 import sys
 from doctest import DocTestSuite, ELLIPSIS
 from math import pi, isnan
+import numpy as np
 
 from sgp4.earth_gravity import wgs72
 from sgp4.ext import invjday, newtonnu, rv2coe
@@ -112,10 +113,10 @@ class Tests(TestCase):
         # Exercise the newtonnu() code path with asinh() to see whether
         # we can replace it with the one from Python's math module.
 
-        self.assertEqual(newtonnu(1.0, 2.9),   # parabolic
+        np.testing.assert_allclose(newtonnu(1.0, 2.9),   # parabolic
                          (8.238092752965605, 194.60069989482898))
 
-        self.assertEqual(newtonnu(1.1, 2.7),   # hyperbolic
+        np.testing.assert_allclose(newtonnu(1.1, 2.7),   # hyperbolic
                          (4.262200676156417, 34.76134082028372))
 
     def test_good_tle_checksum(self):
